src/main/java/ch/njol/util/coll/CollectionUtils.java

```diff
@@ -419,6 +420,30 @@ public static boolean isSubset(final @Nullable Object[] set, final @Nullable Obj
 		}
 		return r;
 	}
+
+	/**
+	 * Converts an array to a non-null array of the specified class.
+	 * Uses registered {@link ch.njol.skript.registrations.Converters} to convert.
+	 * Throws a {@link ClassCastException} if one of {@code original}'s
+	 * elements is a not an instance of {@code to}
+	 *
+	 * @param original The array to convert
+	 * @param to       What to convert {@code original} to
+	 * @return {@code original} converted to an array of {@code to}
+	 */
+	@SuppressWarnings("unchecked")
+	public final static <T> T[] convertArray(Object[] original, Class<T> to) throws ClassCastException {
```

Comment:
This is the exact same thing as Converters#convertArray, isn't it?
https://github.com/bensku/Skript/blob/0bac255ce007c725198241d0a06d3ad879b88272/src/main/java/ch/njol/skript/registrations/Converters.java#L177-L189

Reply:
no, converters#convertArray ignores failure. i could check the length instead i guess but i feel that a method that throws an exception is generally more acceptable when you want to be strict about it